test/game_statistics_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'minitest/autorun'
```

Comment:
This set up looks good, we are moving forward with a similar set up for the other classes, and then we can start creating our methods.